### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by brentongunning**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ $ npm test
 | errors/spec.js | ![done](https://i.imgur.com/RXSkZTD.png "Done") | ![not done](https://i.imgur.com/MleS2Jt.png "Not done") | ![not done](https://i.imgur.com/MleS2Jt.png "Not done") |
 | hdprivatekey.js | ![done](https://i.imgur.com/RXSkZTD.png "Done") | ![not done](https://i.imgur.com/MleS2Jt.png "Not done") | ![not done](https://i.imgur.com/MleS2Jt.png "Not done") |
 | hdpublickey.js | ![done](https://i.imgur.com/RXSkZTD.png "Done") | ![not done](https://i.imgur.com/MleS2Jt.png "Not done") | ![not done](https://i.imgur.com/MleS2Jt.png "Not done") |
-| message.js | ![done](https://i.imgur.com/RXSkZTD.png "Done") | ![done](https://i.imgur.com/RXSkZTD.png "Not done") | ![not done](https://i.imgur.com/MleS2Jt.png "Not done") |
+| message.js | ![done](https://i.imgur.com/RXSkZTD.png "Done") | ![not done](https://i.imgur.com/MleS2Jt.png "Not done") | ![not done](https://i.imgur.com/MleS2Jt.png "Not done") |
```
Comment (brentongunning) [inline README.md]: Revisit random.js

Comment (brentongunning) [reply README.md]: Update message.js in the template file.

Comment (brentongunning) [reply README.md]: See if we can remove the template file

**Comment by ClemensLey** [comment]
LGTM

**Pull request merged**